buildchain/buildchain/constants.py

```diff
@@ -19,6 +19,7 @@
 # URLs of the main container repositories.
 CALICO_REPOSITORY     : str = 'quay.io/calico'
 COREOS_REPOSITORY     : str = 'quay.io/coreos'
+DEX_REPOSITORY        : str = 'quay.io/dexidp'
```

Comment:
Squash this with the "Deploy Dex" commit, there's no need to have a separate commit (if you reverted "Deploy Dex', then this commit still present would mean we ship unneeded container images).